[2025-03-27T13:05:03.041+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.create_destination_table manual__2025-03-27T12:05:02.065240+00:00 [queued]>
[2025-03-27T13:05:03.047+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.create_destination_table manual__2025-03-27T12:05:02.065240+00:00 [queued]>
[2025-03-27T13:05:03.047+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-27T13:05:03.059+0100] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): create_destination_table> on 2025-03-27 12:05:02.065240+00:00
[2025-03-27T13:05:03.062+0100] {standard_task_runner.py:60} INFO - Started process 5208 to run task
[2025-03-27T13:05:03.066+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'test_dag', 'create_destination_table', 'manual__2025-03-27T12:05:02.065240+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpy20fau5o']
[2025-03-27T13:05:03.069+0100] {standard_task_runner.py:88} INFO - Job 89: Subtask create_destination_table
[2025-03-27T13:05:03.128+0100] {task_command.py:423} INFO - Running <TaskInstance: test_dag.create_destination_table manual__2025-03-27T12:05:02.065240+00:00 [running]> on host LAPTOP-A4DHQTS9.
[2025-03-27T13:05:03.205+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='create_destination_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T12:05:02.065240+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-27T12:05:02.065240+00:00'
[2025-03-27T13:05:03.215+0100] {base.py:83} INFO - Using connection ID 'source_db' for task execution.
[2025-03-27T13:05:03.221+0100] {sql.py:450} INFO - Running statement: 
        SELECT column_name, data_type
        FROM information_schema.columns
        WHERE table_name = 'prog3';
    , parameters: None
[2025-03-27T13:05:03.226+0100] {sql.py:459} INFO - Rows affected: 4
[2025-03-27T13:05:03.234+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-27T13:05:03.240+0100] {sql.py:450} INFO - Running statement: 
        CREATE SCHEMA IF NOT EXISTS staging;
        CREATE TABLE IF NOT EXISTS staging.stg_prog3 ("uid" character varying, "lastupdated" timestamp without time zone, "Area (H)" text, "farmers" text);
    , parameters: None
[2025-03-27T13:05:03.241+0100] {python.py:201} INFO - Done. Returned value was: None
[2025-03-27T13:05:03.251+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=create_destination_table, execution_date=20250327T120502, start_date=20250327T120503, end_date=20250327T120503
[2025-03-27T13:05:03.279+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-27T13:05:03.299+0100] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
